jest.transform.js

```diff
@@ -0,0 +1,3 @@
+module.exports = require('babel-jest').createTransformer({
+  presets: ['env', 'react', 'stage-1'], // update this if we need more preprocessors to run tests in Jest
```

Comment:
any update to this comment? should it be removed?

Reply:
nah should prop remove it, good call 👍